crate/kmip/src/kmip/kmip_operations.rs

```diff
@@ -1069,6 +1069,11 @@ pub struct ReKeyKeyPair {
     // Storage Mask selections that are permissible for the new Public Key object.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub public_protection_storage_masks: Option<ProtectionStorageMasks>,
+
+    // Specifies the action to perform during the rekeying
+    #[serde(skip_serializing_if = "Option::is_none")]
+    // Defaults to RotateAttributes
```

Comment:
Defaults to nothing if `None`. The `Default` of `ReKeyKeyPairAction` is `RotateAttributes`, but this is a different matter. Why did you feel the need to specify it here?

Reply:
in the previous versions, ReKeyKeyPair requests were only doing attribute rotations so this is the default behavior to keep backward compability.